focus-areas/security/openssf-best-practices.md

```diff
@@ -1,41 +1,67 @@
 
-As from the [OpenSSF Best Practices Badging Page](https://bestpractices.coreinfrastructure.org/en): The Linux Foundation OpenSSF Best Practices badge is a way for open source projects to show that they follow best practices. Projects can voluntarily self-certify, at no cost, by using this web application to explain how they follow each best practice. Projects receive the passing badge if they meet all of the required criteria.
+The badge focuses on cybersecurity in open source software, helping to ensure that projects follow best practices for producing secure, high-quality software.
 
-## Objectives
+The **OpenSSF Best Practices Badge**:
+- Indicates a project's compliance with "open source project best practices" as defined by the Linux Foundation’s core infrastructure initiative.
+- Encourages open source projects to produce secure software by adhering to established best practices.
+- Enables consumers to assess which open source projects are more likely to produce secure, reliable software based on their compliance with these best practices.
 
-As from the [OpenSSF Best Practices Badging Page](https://bestpractices.coreinfrastructure.org/en): OpenSSF badging indicates a project’s level of compliance with “open source project best practices" as defined by the Linux Foundation's core infrastructure initiative, which focuses on CyberSecurity in open source software. The goal of the program is to encourage projects to produce better more secure software and allow others to determine if a project is following best practices.
+## Want to Know More?
 
-Consumers of the badge can quickly assess which open source projects are following best practices and as a result are more likely to produce higher-quality secure software.
+<span markdown="1"><details>
+<summary>Click to read more about this metric.</summary>
 
-## Implementation
-*The usage and dissemination of health metrics may lead to privacy violations. Organizations may be exposed to risks. These risks may flow from compliance with the GDPR in the EU, with state law in the US, or with other law. There may also be contractual risks flowing from terms of service for data providers such as GitHub and GitLab. The usage of metrics must be examined for risk and potential data ethics problems. Please see [CHAOSS Data Ethics document](https://github.com/chaoss/community/blob/main/data-use-statement.md) for additional guidance.*
-
-See [OpenSSF’s API documentation](https://github.com/coreinfrastructure/best-practices-badge/blob/master/doc/api.md) for relevant information.
+### Data Collection Strategies
+As gotten from the [OpenSSF Best Practices Badging Page](https://bestpractices.coreinfrastructure.org/en), a project’s OpenSSF Best Practices status can be assessed when:

```

Comment:
Can you remove the word "gotten" and change the URL to this: https://www.bestpractices.dev/en. Thank you!

Reply:
Done ✔️